COLLADAMaya/src/COLLADAMayaControllerExporter.cpp

```diff
@@ -555,15 +583,17 @@ namespace COLLADAMaya
                 SceneElement* foundElement = mDocumentExporter->getSceneGraph()->findElement ( pathes[i] );
 				if (foundElement != NULL && !result)
 				{
-					foundElement->addSkeletonURI(skeletonUri);
-					UppersceneElement = sceneElement;
+					
+					JointRootElement = searchJointRootElement(sceneElement);
+					COLLADASW::URI skeletonRootUri(visualSceneExporter->getSceneElementURI(JointRootElement));
+					foundElement->addSkeletonURI(skeletonRootUri);
```

Comment:
Okay then this needs refinement. The skeleton URI can refer to a `<node>` element of either type, not only `JOINT`. It's the root of the search domain, not necessarily an influence itself.